controllers/presentationController.js

```diff
@@ -11,31 +11,69 @@ async function getAllPresentations(req, res) {
     if (!allPresentations) {
       res
         .status(404)
-        .send({ result: "error", error: "Presentations not found" });
+        .json({ message: "No presentations found created by the user" });
       return;
     }
 
     res.json(allPresentations);
   } catch (err) {
-    res.status(500).send({ result: "error", error: "Internal Server Error" });
+    res.status(500).json({ result: "error", error: "Internal Server Error" });
   }
 }
 
-// async function createPresentation(req, res, next) {
-//   // create
-// }
+async function createPresentation(req, res) {
+  const { title } = req.body;
 
-// async function savePresentation(req, res, next) {
-//   // save
-// }
+  try {
+    const newPresentation = await Presentation.create(title);
+
+    res.status(201).json(newPresentation);
+  } catch (err) {
+    res.status(500).json({ result: "error", error: "Internal Server Error" });
+  }
+}
+
+async function deletePresentation(req, res) {
+  const { id } = req.params;
 
-// async function deletePresentation(req, res, next) {
-//   // delete
-// }
+  try {
+    const deletedPresentation = await Presentation.findByIdAndDelete(id);
+
+    if (!deletedPresentation) {
+      res.status(404).json({ message: "No presentation found to delete" });
```

Comment:
```suggestion
      res.status(404).json({ result: "error", message: "No presentation found to delete" });
```
이 부분에도 result를 추가하는 게 좋을 것 같습니다 🤓